pkg/reconciler/managed/api.go

```diff
@@ -172,12 +172,18 @@ func NewRetryingCriticalAnnotationUpdater(c client.Client) *RetryingCriticalAnno
 // reset to their current state according to the API server.
```

Comment:
Perhaps change this comment from "pending changes _are_ reset" to "pending changes _may be_ reset".

Reply:
done